provisioning/transport/http/typings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "azure-iot-provisioning-http",
+  "main": "index.d.ts",
+  "dependencies": {},
+  "globalDependencies": {
+    "node": "registry:dt/node#6.0.0+20160813124416"
```

Comment:
>node": "registry:dt/node#6.0.0+20160813124416" [](start = 5, length = 46)

no need for a typings file if you have @types/node installed in your dependencies.

Reply:
You'll have to explain this to me later.

---
In reply to: [142263803](https://github.com/Azure/azure-iot-sdk-node/pull/99#discussion_r142263803) [](ancestors = 142263803)